.eslintrc.js

```diff
@@ -1,17 +1,11 @@
 'use strict';
 
-const fs = require('fs');
-const path = require('path');
-const PACKAGE_NAME = require('./package').name;
-const SYMLINK_LOCATION = path.join(__dirname, 'node_modules', PACKAGE_NAME);
-
-// Symlink node_modules/{package name} to this directory so that ESLint resolves this plugin name correctly.
-if (!fs.existsSync(SYMLINK_LOCATION)) {
-  fs.symlinkSync(__dirname, SYMLINK_LOCATION);
-}
+// Register ourselves as a plugin to avoid `node_modules` trickery.
+const Plugins = require('eslint/lib/config/plugins');
+Plugins.define('prettier', require('.'));
```

Comment:
> Hmm, I hadn't realized modifying node_modules breaks yarn

It causes `yarn check` to fail. Also, it'll break a new yarn feature I'll be upstreaming soon: "watchman-based check". Basically, you get instant (sub ~100ms) full `node_modules` verification, which lets use put `yarn` in front of every js script w/o having to worry about stale modules.